src/test/java/com/fasterxml/jackson/core/write/UTF8GeneratorTest.java

```diff
@@ -183,6 +188,43 @@ public void testWriteLargeIntegerStringWriter() throws Exception
         }
     }
 
+    public void testWriteLargeIntegerDataOutput() throws Exception
```

Comment:
Maybe add a brief comment on background, testing that large/long numbers can be written fine, without changing default settings?

Reply:
Actually n/m, I can add.